modules/auxiliary/scanner/portscan/ack.rb

```diff
@@ -43,16 +45,26 @@ def run_batch_size
   end
 
   def run_batch(hosts)
-    open_pcap
-
-    pcap = self.capture
 
     ports = Rex::Socket.portspec_crack(datastore['PORTS'])
-
     if ports.empty?
       raise Msf::OptionValidateError.new(['PORTS'])
     end
 
+    jitter_value = datastore['JITTER'].to_i
```

Comment:
Is it necessary to add the following code to [**```/lib/msf/core/auxiliary/scanner.rb```**](https://github.com/rapid7/metasploit-framework/blob/master/lib/msf/core/auxiliary/scanner.rb)


```
  register_advanced_options([
    OptBool.new('ShowProgress', [true, 'Display progress messages during a scan', true]),
    OptInt.new('ShowProgressPercent', [true, 'The interval in percent that progress should be shown', 10]),
    OptInt.new('DELAY', [true, "The delay between connections, per thread, in milliseconds", 0]),
    OptInt.new('JITTER', [true, "The delay jitter factor (maximum value by which to +/- DELAY) in milliseconds.", 0])
  ], Auxiliary::Scanner)
```